ext/winevt/winevt_c.h

```diff
@@ -73,6 +75,12 @@ struct WinevtSubscribe
   DWORD count;
   DWORD flags;
   BOOL tailing;
+  DWORD rateLimit;
+  time_t lastTime;
+  DWORD previousRate;
+  DWORD currentRate;
+  BOOL (*limit_check_handler)(struct WinevtSubscribe*);
+  void (*state_handler)(struct WinevtSubscribe*, ULONG count);
```

Comment:
These function pointers aren't needed anymore.
The functions should be called directly.